themes/custom/az_barrio/css/style.css

```diff
@@ -368,6 +368,25 @@ td.menu-disabled {
   }
 }
 
+.breadcrumb-item+.breadcrumb-item:before {
```

Comment:
These `.breadcrumb` styles actually live in AZ Bootstrap, so any styles changes should probably go through Bootstrap rather than living in Quickstart.